packages/express-openapi/README.md

```diff
@@ -1051,6 +1102,88 @@ POST.apiDoc = {
 };
 ```
 
+#### TypeScript Example of Global Response Validation
```

Comment:
i don't feel we need a typescript example.  most users will know how to port it to typescript.

Reply:
I'm wondering if you would consider keeping it. Here's my logic:

1. Can the same be said for the existing TypeScript example? If anything, the library itself has awesome and extensive type definitions, so it seems like TypeScript is a first-class citizen.
2. If the examples are here to address a potentially common use case, does the benefit of reducing friction outweigh the addition of ~81 lines in a document which is already longer than 1100 lines? Maybe the documentation would be served by a refactor into multiple files. But even if a developer _can_ port it and refactor to fit types/interfaces/etc., should they _have to_?